src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -700,6 +700,17 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
 
                     return style.listStyleImage;
                 }
+            },
+
+            // xhr
+            responseURL: {
+                condition: domUtils.isXMLHttpRequest,
```

Comment:
Do we really need code instrumentation here? Can't we just define property on xhr object?